test/unit/app/jobs/test_job_wrapper.py

```diff
@@ -120,7 +120,7 @@ def set_compute_environment(self, *args, **kwds):
         pass
```

Comment:
The `populate_interactivetools` method at line 116-117 above can now be removed.